src/silx/gui/plot/items/_roi_base.py

```diff
@@ -97,7 +97,7 @@ def _updated(self, event=None, checkVisibility=True):
         """
         self.sigItemChanged.emit(event)
 
-    def contains(self, position):
+    def contains(self, position: tuple[float, float]) -> bool:
         """Returns True if the `position` is in this ROI.
 
         :param tuple[float,float] position: position to check
```

Comment:
```suggestion
        :param position: position to check
```

`:rtype: bool` below should also be removed